src/components/auth/AuthForm.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button } from 'primereact/button';
+import { InputText } from 'primereact/inputtext';
+import { Password } from 'primereact/password';
+import { useState } from 'react';
+import { loginUser } from './service';
+import '../../styles/components/Auth.styles.scss';
```

Comment:
```suggestion
import styles from  '@styles/components/AuthForm.styles.scss';
```
В новых ветках добавил инструмент для избегания лесенок в импортах. Теперь нужно вот так импортировать

Reply:
Название классов передаем через объект styles

Reply:
У меня вылетали при таком определении ошибки, попробую ещё раз